service/src/integrationTest/kotlin/fi/espoo/evaka/FullApplicationTest.kt

```diff
@@ -59,7 +60,7 @@ abstract class FullApplicationTest {
 
     @Autowired
     protected lateinit var env: Environment
-    @Autowired
+    @SpyBean
```

Comment:
Do we really need to use a mock here? :thinking: 
It's a big change, especially since it affects all tests that use `FullApplicationTest`